app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -28,6 +29,10 @@ def host_subscriptions_names(host)
         host.subscriptions.map(&:name)
       end
 
+      def host_redhat_subscriptions(host)
+        host.subscriptions.where.not(:support_level => nil).map(&:name)
```

Comment:
We can improve this check to find a redhat subscription by adding a scope here:

https://github.com/Katello/katello/blob/d6cb33076d4606676086c0159282aaef5022fda9/app/models/katello/subscription.rb#L13

It would look like this:
```
scope :redhat, ->() { joins(:products).merge(Katello::Product.redhat).distinct }
```

So you can call:

```
host.subscriptions.redhat.pluck(:name)
```

This generates an optimal query, and we can re-use the new scope in other areas of the code in the future :)